core/classes/DTO/IntegrationData.php

```diff
@@ -5,15 +5,15 @@ class IntegrationData {
     public int $id;
     public string $name;
     public bool $enabled;
-    public bool $can_unlink;
+    public bool $can_be_unlink;
```

Comment:
This change is not something we will consider merging - changing a column of the user database is not something we want to do. Your initial commit was good, could you revert back to just that?

Reply:
Then that's the way it is? What's wrong with that?

Reply:
Sorry, I don't understand what you mean 😄 
`can_unlink` is valid enough English for it's use case here - we will not be changing it. Your first commit was good so if you'd revert to that we can merge it 🙂

Reply:
> Sorry, I don't understand what you mean 😄 `can_unlink` is valid enough English for it's use case here - we will not be changing it.

Dude, can_unlink can be used as a permission name! Here it is the question whether an integration can be unlinked! So: Can be unlink = can_be_unlink

Reply:
"can be unlink" is not valid English. "can be unlinked" would be, but the existing name is also fine. Please keep to one change per PR :)

Reply:
> "can be unlink" is not valid English. "can be unlinked" would be, but the existing name is also fine. Please keep to one change per PR :)

As I said before

> Dude, can_unlink can be used as a permission name! Here it is the question whether an integration can be unlinked!


And one change per PR? Sorry, but... Stupid? How many PRs do you want? Literally a few hundred thousand?

Reply:
> And one change per PR? Sorry, but... Stupid? How many PRs do you want? Literally a few hundred thousand?

Yes, makes it easier to revert them

> As I said before

Your updated term was actually wrong. Though we are not planning on changing the name of the column in the database as this is just simply unnecessary and would be a major breaking change. It is fine the way it is right now. So that's why we're asking to revert that change so we can just fix the redundant language term.

Reply:
We're talking about the `adding_server` language term. The `can unlink` changes will not be merged. So that's why we are asking to revert those changes.

Reply:
All we're asking is if you can just revert the changes made related to the `can_unlink` string. As previously mentioned the way it is currently worded `can_unlink` is completely valid the way it is used right now. Your updated string is actually not correct english. Additionally, we are not planning on changing that database column. 

So if you can *just* include the original changes you made (eg: the `NEW_SERVER` one) then we can merge this. We will not be merging this PR in it's current state due to the can_unlink change.

Reply:
Already, yes, I have not changed the title above! The **_Update_** as the title was intentional! And whether you merge the two, or both separately, this major breaking change would be either way! And `can_unlink` is simply wrong, because this sounds like an permission, but not an action!

In German this is called _Kann entlinken_! However, the _can_unlink_ refers to the integration, not to the user

Reply:
> There won't be a breaking change if we don't do the change, which we aren't planning on doing anyways. can_unlink is completely correct in it's current context. It indicates if the integration can be unlinked (true or false). So I don't see anything wrong with it? It can be used in both contexts.

Exactly not! If we keep it in this context, it sounds as if the integration could unlink other things! An integration can not do this! _can_unlink_ is related to the user, that is, can be unlinked

Reply:
From me! Fixed

5659cf2